mars/dataframe/core.py

```diff
@@ -1021,6 +1021,18 @@ def index(self):
     def axes(self):
         return [self.index]
 
+    """
+    Property .empty STARTS
```

Comment:
Usually, we use git blame to show the the revision and author of code, comments are not necessary.

Reply:
Sure then :D